src/app/page.tsx

```diff
@@ -1,86 +1,73 @@
-"use client";
-import React, {useState} from 'react';
-import styles from './styles.module.css';
-import InputTodo from "@/app/components/InputTodo";
+import Todo from "@/components/Todo";
 
-const Todo = () => {
-    const [todoText, setTodoText] = useState("");
-    const [incompleteTodos, setIncompleteTodos] = useState(["TODO1", "TODO2"]);
-    const [completeTodos, setCompleteTodos] = useState(["TODO1", "TODO2"]);
+const Home = () => {
+  return <Todo />;
+};
 
-    const onChangeTodoText = (event:React.ChangeEvent<HTMLInputElement>) => setTodoText(event.target.value);
+export default Home;
```

Comment:
あっています。

ちなみにですが、こういった書き方もできます。



```suggestion
export default Todo;
```